### ISSUE TITLE
IVOM/KLI debugging

### ISSUE THREAD

**Issue by kdm9** [Adamtaranto/frisk#41]
I'll attach an example fasta file here. Perhaps do `1<= k <= 4` as that will fit nicely on a screen.

K

**Comment by kdm9** [comment]
Better late than never. 100kb 30%GC, 10kb 80%GC, 100kb 30%gc. Purely random seqs.
The FASTA is [test.fa.gz](https://github.com/Adamtaranto/frisk/files/268128/test.fa.gz)

**Comment by kdm9** [comment]
Any progress on this @Adamtaranto

**Comment by Adamtaranto** [comment]
Yep, @kdmurray91, I'll push up the min_ivom_kld branch later this evening.
